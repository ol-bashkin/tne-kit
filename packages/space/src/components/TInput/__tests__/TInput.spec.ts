import { mount } from "@vue/test-utils";
import TInput from "../TInput.vue";

describe("WInput.vue", () => {
  test("should have an t-input class", () => {
    const wrapper = mount(TInput, {
      propsData: {
        modelValue: "test",
      },
    });
    expect(wrapper.classes()).toContain("t-input");
    expect(wrapper.html()).toMatchSnapshot();
  });
});
