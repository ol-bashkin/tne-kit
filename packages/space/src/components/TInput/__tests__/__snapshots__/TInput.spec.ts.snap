// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WInput.vue should have an t-input class 1`] = `
<div class="t-input t-input--lg secondary--base">
  <input class="t-text font-primary text-primary--base" type="text">
</div>
`;
